lms/djangoapps/ora_staff_grader/tests/test_serializers.py

```diff
@@ -360,16 +355,22 @@ def test_serializer_output(self):
             expected_submissions_data[submission_id] = serialized_data
 
         # Check that each of the sub-serializers assembles data correctly
-        assert output_data['courseMetadata'] == CourseMetadataSerializer(self.mock_course_metadata).data
-        assert output_data['oraMetadata'] == OpenResponseMetadataSerializer(self.mock_ora_instance).data
-        assert output_data['submissions'] == expected_submissions_data
+        assert (
```

Comment:
i personally hate paren line breaks when they aren't necessary. Could we pull these into variables to prevent the line breaks?